lib/routes.js

```diff
@@ -0,0 +1,15 @@
+var routes = {
+  method: "GET",
+  path:   "/version",
+  config: {
+    state: {
+      parse:      true,
+      failAction: "log"
```

Comment:
Bizarre; I can't reproduce this. What version of Node are you on?

This is my diff:

https://gist.github.com/nhunzaker/e920352cd9b0a3fe944c

Do I have this right?